src/icon-tab-bar.scss

```diff
@@ -1127,4 +1203,49 @@ $fd-icon-tab-bar-semantic-values: (
       @include fd-apply-responsive-paddings();
     }
   }
+
+  .#{block}-dnd-preview {
+    background: var(--sapObjectHeader_Background, #fff);
```

Comment:
Any example of this?

Reply:
I added some classes for drag and drop actions. I don't know how to show it in examples without js. Do you have any ideas?